### PR TITLE
21315 - Upload Director's Affidavit

### PR DESCRIPTION
**Pull request by ArwenQin**
*Issue #:* /bcgov/entity#21315

*Description of changes:*
- Added Upload Director's Affidavit component for manual input
- Restructured code, added a new sub-component UploadAffidavit

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the bcrs-entities-create-ui license (Apache 2.0).

### CODE REVIEW

**Comment by ArwenQin** [comment]
I will add unit tests.

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://business-create-dev--pr-697-tnyj58ql.web.app

File: src/components/ContinuationIn/ExtraproRegistration.vue
Comment (severinbeauvais) [inline src/components/ContinuationIn/ExtraproRegistration.vue:236]: I'd like you to keep the v-row and v-col in here since it's part of the layout in this component.

If you keep the v-row and v-col in the sub-component then that's how it must be used in the future... but we might want it in a different layout. So, remove it from there.

The principle I'm following here is scope from a layout point of view.

File: src/components/ContinuationIn/ExtraproRegistration.vue
Comment (severinbeauvais) [inline src/components/ContinuationIn/ExtraproRegistration.vue:394]: Do you still need DocumentMixin in this component?

File: src/components/ContinuationIn/ManualBusinessInfo.vue
Comment (severinbeauvais) [inline src/components/ContinuationIn/ManualBusinessInfo.vue:237]: I think you don't need DocumentMixin or PdfPageSize in here.

File: src/components/ContinuationIn/ExtraproRegistration.vue
Comment (severinbeauvais) [reply src/components/ContinuationIn/ExtraproRegistration.vue:394]: What about PdfPageSize?

File: src/components/ContinuationIn/ManualBusinessInfo.vue
Comment (severinbeauvais) [reply src/components/ContinuationIn/ManualBusinessInfo.vue:237]: I think you don't need getKeycloakGuid either.

Comment (severinbeauvais) [reply src/components/ContinuationIn/ManualBusinessInfo.vue:237]: Or customErrorMessage.

File: src/components/ContinuationIn/UploadAffidavit.vue
Comment (severinbeauvais) [inline src/components/ContinuationIn/UploadAffidavit.vue:67]: You won't need this when you move the v-row back into the parent component.

File: src/components/ContinuationIn/ExtraproRegistration.vue
Comment (ArwenQin) [reply src/components/ContinuationIn/ExtraproRegistration.vue]: Got it! Updated

File: src/components/ContinuationIn/ExtraproRegistration.vue
Comment (ArwenQin) [reply src/components/ContinuationIn/ExtraproRegistration.vue]: Removed unused codes

File: src/components/ContinuationIn/ManualBusinessInfo.vue
Comment (ArwenQin) [reply src/components/ContinuationIn/ManualBusinessInfo.vue]: Removed unused codes

File: src/components/ContinuationIn/UploadAffidavit.vue
Comment (ArwenQin) [reply src/components/ContinuationIn/UploadAffidavit.vue]: Thanks, updated. Moved the v-row back into the parent component.

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://business-create-dev--pr-697-tnyj58ql.web.app

File: src/components/ContinuationIn/ManualBusinessInfo.vue
Comment (severinbeauvais) [inline src/components/ContinuationIn/ManualBusinessInfo.vue:263]: I think you don't need this one any more.

File: src/components/ContinuationIn/UploadAffidavit.vue
Comment (severinbeauvais) [inline src/components/ContinuationIn/UploadAffidavit.vue:85]: Update comment, please.

The comment on line 50 looks good 👍

**Review by severinbeauvais — APPROVED** [review]
Looks great. Very clean :)

I'm approving now as I'm leaving early today. I trust you to update some unit tests. Wait for another approval and then this PR can be merged.

File: src/components/ContinuationIn/ManualBusinessInfo.vue
Comment (ArwenQin) [reply src/components/ContinuationIn/ManualBusinessInfo.vue]: Removed

File: src/components/ContinuationIn/UploadAffidavit.vue
Comment (ArwenQin) [reply src/components/ContinuationIn/UploadAffidavit.vue]: Updated

**Comment by ArwenQin** [comment]
> Looks great. Very clean :)
> 
> I'm approving now as I'm leaving early today. I trust you to update some unit tests. Wait for another approval and then this PR can be merged.

Thank you. Unit tests are added.

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://business-create-dev--pr-697-tnyj58ql.web.app

**Pull request merged**
